core/templates/services/context.service.spec.ts

```diff
@@ -266,5 +315,15 @@ describe('Context service', () => {
       expect(ecs.getPageContext()).toBe('other');
     }
     );
+
+    it('should retrive exploration preview mode', () => {
+      expect(ecs.getEditorTabContext()).toBe('preview');
+    });
+
+    it('should retrieve entity id and type', () => {
```

Comment:
Isn't this test related to setting/getting a custom entity type?

Reply:
Done!